_sass/_layouts/_home.scss

```diff
@@ -1,6 +1,6 @@
 body {
   background: $body-background-color;
-  font-family: 'Vollkorn', serif;
+  font-family: 'Source Sans Pro', sans-serif;
```

Comment:
Why is the main font changed?

Reply:
While Using ```'Vollkorn', serif;``` 1 is seems like I, And I searched better font ```'Source Sans Pro', sans-serif;``` is looks good for me. That's why I changed the font. 

<img width="804" alt="screen shot 2018-06-03 at 8 51 12 pm" src="https://user-images.githubusercontent.com/8276878/40887979-6543086a-6770-11e8-8838-55f8391bc76e.png">